packages/@angular/cli/models/webpack-configs/browser.ts

```diff
@@ -88,18 +88,19 @@ export function getBrowserConfig(wco: WebpackConfigOptions) {
       crossOriginLoading: buildOptions.subresourceIntegrity ? 'anonymous' : false
     },
     plugins: [
-      new HtmlWebpackPlugin({
-        template: path.resolve(appRoot, appConfig.index),
-        filename: path.resolve(buildOptions.outputPath, appConfig.index),
-        chunksSortMode: packageChunkSort(appConfig),
-        excludeChunks: lazyChunks,
-        xhtml: true,
-        minify: buildOptions.target === 'production' ? {
-          caseSensitive: true,
-          collapseWhitespace: true,
-          keepClosingSlash: true
-        } : false
-      }),
+      // Not yet supported in webpack 4.
+      // new HtmlWebpackPlugin({
```

Comment:
I guess I do not have `index.html` in build because of the following code?

Reply:
Correct, we're still waiting for plugins to be updated. At the moment this PR is not ready.

Reply:
Thank you, looks like we are expecting #816 to be resolved